doc/en/components/grids/_shared/editing.md

```diff
@@ -159,11 +159,12 @@ igRegisterScript("SortingHandler", SortingHandler, false);
 
 <!-- React -->
 ```tsx
-function onSorting(grid: {ComponentName}, event: IgrSortingEventArgs) {
+function onSorting(args: IgrSortingEventArgs) {
+    const grid = args.target as IgrGrid;
```

Comment:
Casting here as `IgrGrid` breaks the generation of the snippet for Hgrid and Treegrid.

Reply:
casted as component selector to so it will change to the relevant component based on topic now